build/SharedFrameworkOnly.props

```diff
@@ -11,6 +11,10 @@
     <SharedFrameworkAndPackage Include="Microsoft.AspNetCore.JsonPatch" />
     <SharedFrameworkAndPackage Include="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" />
 
+    <!-- Assemblies required by components -->
```

Comment:
Might be clearer to specify they are here because they are needed on *WebAssembly*, not just by components in general.